databroker/broker.py

```diff
@@ -577,6 +577,46 @@ def process(self, headers, func, fields=None, fill=False):
 
     get_fields = staticmethod(get_fields)  # for convenience
 
+    def export(headers, mds, fs, new_root=None):
```

Comment:
We can simplify this signature by just passing in a `Broker` instance that has `mds` and `fs` as attributes. I would suggest the signature: `export(headers, db, new_root=None`)